testsuite/layers/pom.xml

```diff
@@ -2161,6 +2381,8 @@
                                                 <layer>discovery</layer>
                                                 <layer>ee</layer>
                                                 <layer>ee-security</layer>
+                                                <layer>ejb</layer>
+                                                <layer>ejb-lite</layer>
```

Comment:
@ehugonne I can't see the `remote-activemq` added to `all-layers-provisioning-full` as written in the TP (https://gitlab.mw.lab.eng.bos.redhat.com/jbossqe-eap/test-plans/merge_requests/392/diffs#1135f3443166464178d2031722561a56e1bdc307_0_105)